app-editors/pluma-plugins/metadata.xml

```diff
@@ -9,13 +9,6 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
-<use>
-	<flag name="bracketcompletion">Auto complete bracket pairs</flag>
-	<flag name="codecomment">Comment code blocks</flag>
```

Comment:
Presumably this should be squashed into the commit that removed the ebuild with those flags...